### PR TITLE
Move artifact chown inside docker to avoid sudo

### PR DESCRIPTION
**Pull request by oddstr13**
I found the sudo prompt for moving the files into the final location annoying.

### CODE REVIEW

File: deployment/ubuntu-package-armhf/docker-build.sh [hunk 1]
```diff
@@ -18,3 +18,4 @@ dpkg-buildpackage -us -uc -aarmhf
 # Move the artifacts out
 mkdir -p ${ARTIFACT_DIR}/deb
 mv /jellyfin_* ${ARTIFACT_DIR}/deb/
+chown -Rc $(stat -c %u:%g ${ARTIFACT_DIR}) ${ARTIFACT_DIR}
```
Comment (EraYaN) [inline deployment/ubuntu-package-armhf/docker-build.sh]: Do these `%u` and `%g` reflect the same user as `${current_user}`?

Comment (oddstr13) [reply deployment/ubuntu-package-armhf/docker-build.sh]: Yes.
What this does, is taking the `UID` and `GID` of `${ARTIFACT_DIR}` (which is created outside of docker), and applying it recursively to the content.

**Comment by oddstr13** [comment]
rebased on top of `jellyfin:release-10.3.z` tip, added arm64 debian and ubuntu changes

**Comment by oddstr13** [comment]
Please ignore that last force-push.
Also, thanks to whoever linked https://dangitgit.com/ in the chat.

**Pull request merged**